tests/unit/conftest.py

```diff
@@ -179,3 +176,12 @@ def home_temporary_file():
         home_temporary_file,
     )
     return temp_file
+
+
+@pytest.fixture()
+def mock_verify_network(fake_process):
```

Comment:
To get **very** technical, this is neither a fake nor a mock, but a stub. (It returns a real, but predefined, value.)

(But also, it's not like this distinction is actually particularly useful here.)